plugins/webapi/info/plugin.go

```diff
@@ -92,9 +92,9 @@ type Response struct {
 	// public key of the node encoded in base58
 	PublicKey string `json:"publickey,omitempty"`
```

Comment:
Shouldn't it also be `json:"publicKey,omitempty"` if everything else is small camelCase?